src/Symfony/Bridge/Doctrine/Messenger/AbstractDoctrineMiddleware.php

```diff
@@ -25,8 +25,8 @@
  */
 abstract class AbstractDoctrineMiddleware implements MiddlewareInterface
 {
-    protected $managerRegistry;
-    protected $entityManagerName;
+    protected ManagerRegistry $managerRegistry;
+    protected ?string $entityManagerName;
```

Comment:
Caution: While this class is `@internal`, its children are not. This is still a BC break.

Reply:
right reverted